center/src/main/resources/application-release.yml

```diff
@@ -8,6 +8,7 @@ spring:
             authorization-uri: https://login.microsoftonline.com/${MICROSOFT_PROVIDER_TENANT_ID}/oauth2/v2.0/authorize
             token-uri: https://login.microsoftonline.com/${MICROSOFT_PROVIDER_TENANT_ID}/oauth2/v2.0/token
             jwk-set-uri: https://login.microsoftonline.com/${MICROSOFT_PROVIDER_TENANT_ID}/discovery/v2.0/keys
+            photo-uri: https://graph.microsoft.com/v1.0/me/photo/$value
```

Comment:
Could you share why we need to add this uri?

Reply:
It's a misdeleted uri for portal photo requesting, I've checked it in in another PR.